pkgs/by-name/v4/v4l-utils/package.nix

```diff
@@ -62,14 +64,11 @@ stdenv.mkDerivation (finalAttrs: {
     (lib.mesonEnable "gconv" stdenv.hostPlatform.isGnu)
     (lib.mesonEnable "qv4l2" withQt)
     (lib.mesonEnable "qvidcap" withQt)
+    (lib.mesonEnable "bpf" withBPF)
     (lib.mesonOption "udevdir" "${placeholder "out"}/lib/udev")
   ]
   ++ lib.optionals stdenv.hostPlatform.isGnu [
     (lib.mesonOption "gconvsysdir" "${glibc.out}/lib/gconv")
-  ]
-  ++ lib.optionals (stdenv.hostPlatform != stdenv.buildPlatform) [
-    # BPF support fail to cross compile, unable to find `linux/lirc.h`
```

Comment:
@avnik I can't reproduce this failure using `pkgsCross.aarch64-multiplatform.v4l-utils` on x86_64-linux. See https://github.com/NixOS/nixpkgs/pull/429900 for the PR that made this conditional on non-cross.

Reply:
I'm re-enabling BPF support in https://github.com/NixOS/nixpkgs/pull/467632.